### DISCUSSION TITLE
Name?

### DISCUSSION THREAD

**Discussion by cherylli** [webdev-20/shorti-url-shortener#15] [Ideas]
Shorti 
@gschadegg

**Comment by cherylli** [comment]
compressIT
@Kcasa

**Comment by cherylli** [comment]
MicroURL
@Denise-Carlson

**Comment by MaestroDagan** [comment]
> Shorti @gschadegg

Maybe change the repo name to this to?

**Reply by MaestroDagan** [reply]
And change the description so it's more descriptive?
